test/manipulation.test.js

```diff
@@ -329,12 +557,24 @@ describe('manipulation', function () {
         });
     });
 
+    it('should ignore undefined values on updateAttributes (promise variant)', function(done) {
+      person.updateAttributes({'name': 'John', age: undefined})
+      .then(function(p) {
+        return Person.findById(p.id)
+        .then(function(p) {
+          p.name.should.equal('John');
+          p.age.should.equal(15);
+          done();
+        });
+      }).catch(done);
```

Comment:
"ignore undefined values" is not related to callback vs. promise, can we omit this test?